sympy/functions/combinatorial/factorials.py

```diff
@@ -1239,6 +1239,8 @@ def _eval_evalf(self, prec):
     @classmethod
     def _binomial(cls, *args, symbolic=True):
         from sympy import binomial
+        if args[0].is_number and all(x==args[0] for x in args):
```

Comment:
maybe this should go inside the 2nd `if` below so a symbolic results can be given, too:
```
if len(set(args)) == 1:
    return factorial(sum(args))/factorial(args[0])**len(args)
```

Reply:
Okayyy, that'll work, awesome! 😄

Reply:
I'll push it along with changes to the Piecewise part, after you review it.